tools/roslaunch/src/roslaunch/substitution_args.py

```diff
@@ -163,8 +163,7 @@ def _find_executable(resolved, a, args, _context):
     # we try to find the specific executable in libexec via catkin
     # which will search in install/devel space
     full_path = None
-    # don't know why not imported at the top
-    from roslaunch.resource_cache import find_in_workspaces
+    from catkin.find_in_workspaces import find_in_workspaces
     paths = find_in_workspaces(['libexec'], project=args[0], first_matching_workspace_only=True)  # implicitly first_match_only=True
```

Comment:
Can we use cached resource for find_in_package? (not only find_node at https://github.com/ros/ros_comm/pull/682/files#diff-67998749e41c63572aa0901b6715e9a0R251)